src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -272,6 +275,39 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!--
     ============================================================
+                                        _ComputeReferenceAssemblies
+
+    Computes references that are only used at compile-time.
+    ============================================================
+    -->
+  <Target Name="_ComputeReferenceAssemblies">
+
+    <ItemGroup>
+      <_FrameworkReferenceAssemblies Include="@(ReferencePath)"
+                                     Condition="%(ReferencePath.FrameworkFile) == 'true' or
```

Comment:
It is definitely still necessary.  I took this check away and a bunch of the Reference Assemblies (ex. `mscorlib.dll`) were no longer getting picked up.

Debugging into it, by the time it gets to `GenerateDepsFile`, I see `mscorlib` defined like this:

```
C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.6.1\mscorlib.dll
    CopyLocal=false
    FileGroup=FrameworkAssembly
    FrameworkAssembly=mscorlib
    FrameworkFile=true
    FusionName=mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
    ImageRuntime=v4.0.30319
	NuGetIsFrameworkReference=true
    NuGetPackageId=System.Xml.XPath.XDocument
    NuGetPackageVersion=4.3.0
    NuGetSourceType=Package
    OriginalItemSpec=mscorlib
    PackageName=System.Xml.XPath.XDocument
    PackageVersion=4.3.0
    ParentPackage=System.Xml.XPath.XDocument/4.3.0
    ParentTarget=.NETFramework,Version=v4.6.1
    Private=false
    Redist=Microsoft-Windows-CLRCoreComp.4.6.1
    ReferenceAssembly=%(FullPath)
    ReferenceSourceTarget=ResolveAssemblyReference
    ResolvedFrom={TargetFrameworkDirectory}
    Version=4.0.0.0
```

Note this:

```
    NuGetPackageId=System.Xml.XPath.XDocument
    NuGetPackageVersion=4.3.0
    NuGetSourceType=Package
```

No clue who is putting this here, and I didn't take the time to investigate yet.

I'm not sure the best way to check for framework "reference assembly" assemblies, but this was the existing logic, and it didn't feel like a good time changing it.  Especially since I tried changing it and it broke - 😞